terragrunt/aws/alert_compromise/functions/broadcast_alert.py

```diff
@@ -26,7 +26,18 @@ def lambda_handler(event, context):
     message = decoded_event["logEvents"][0]["message"]
 
     # List of items to ignore as these are from GitHub testing
-    ignore_terms = ["dsp-testing", "example.com", "gcntfy-github-test-revoked"]
+    ignore_terms = [
+        "dsp-testing",
+        "example.com",
+        "gcntfy-github-test-revoked",
+        "cds-snc/notification-documentation",
+    ]
+
+    # get the notify_api_key from the environment variable
+    notify_test_api_key = os.environ["notify_test_api_key"]
```

Comment:
For your question about passing the test key as a secret, I think it would be safe just to rely on the `cds-snc/notification-documentation` exclusion you have above and remove the extra secret and full API key.

Reply:
Thanks Pat - you are totally right that whitelisting the documentation repo is sufficient. I think though I will leave the API key in there just in case we want to use it somewhere else, such as unit tests or other places rather than documentation. I think this might come up again and it might be handy to have a whitelisted safe key (especially if I have already done the work)